testutil/singlevm/configuration.yaml

```diff
@@ -9,6 +9,9 @@ configure:
   image_service:
     url: http://glance.example.com
   launcher:
+    compute_net: 192.168.12.1/24
```

Comment:
Can we use the reserved IP subnet 198.51.100.0/24
This is a special network reserved for testing and will not show up in a real world network
See https://en.wikipedia.org/wiki/Reserved_IP_addresses